src/applications/pensions/tests/e2e/helpers/keyboardOnlyHelpers.js

```diff
@@ -382,7 +382,7 @@ export const keyboardTestPage = (page, data) => {
 export const startForm = () => {
   cy.url().should('include', '/introduction');
 
-  cy.tabToElement('.schemaform-start-button, .vads-c-action-link--green');
+  cy.tabToElement('va-link-action');
```

Comment:
These keyboard e2e test changes don't fix the kitchen sink and overflow e2e tests, which aren't run in CI.